examples/ios/HelloWorldSwift/BUILD

```diff
@@ -12,6 +12,40 @@ swift_library(
     data = [
         "Resources/Main.storyboard",
     ],
+    module_name = "Sources",
+)
+
+swift_library(
+    name = "Tests",
+    testonly = True,
+    srcs = [
+        "Tests/Tests.swift",
+    ],
+    deps = [
+        ":Sources",
+    ],
+)
+
+ios_unit_test(
+    name = "UnitTests",
+    minimum_os_version = "8.0",
+    deps = [":Tests"],
+)
```

Comment:
Coverage from this file without the sources exclusion https://gist.github.com/maxwellE/0180250b83dbc351064a7b68057148c6 . We can see that paths are still made relative even with the $PWD change